setup.cfg

```diff
@@ -106,7 +106,7 @@ testing =
 [options.entry_points]
 # CLI entry points to be installed
 console_scripts =
-    gtpyc = gt4py.cli:gtpyc
+    gtpyc = gt4py.cartesian.cli:gtpyc
```

Comment:
We now use gt4pyc as shorthand for `gt4py.cartesian` in imports. I don't know a better naming but the names are very close.

Reply:
yeah, I agree, but wouldn't change now